recipes/parcels/meta.yaml

```diff
@@ -0,0 +1,66 @@
+{% set name = "parcels" %}
+{% set version = "1.0" %}
+{% set sha256 = "d9753572a3f099caf5c17b477753034fe8f714788e588874ade18fd91a573f13" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/oceanparcels/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+  patches:
+      - remove_setuptools-scm_dependency.patch
+
+build:
+  skip: True  # [py3k]
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - setuptools_scm
+  run:
+    - python
+    - cachetools >=1.0.0
+    - cgen
+    - enum34
+    - ffmpeg >=3.2.3,<3.2.6
+    - gcc  # [unix]
+    - m2w64-toolchain  # [win]
+    - matplotlib 2.0.2
+    - netcdf4 >=1.1.9
+    - numpy >=1.9.1
+    - progressbar2
+    - python-dateutil
+    - scipy >=0.16.0
+    - six >=1.10.0
+    - xarray >=0.5.1
+test:
+  imports:
+    - parcels
```

Comment:
Would it be possible to add more import tests?

Reply:
Yes.  I'll add explicit imports of the relevant sub-modules.